.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Doc issue
+about: Create an issue to help us improve
+---
+
+**Before you open an issue**
+
+If the issue is:
+
+- A simple typo or similar correction, you can submit a PR. See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions.
+- A general support question, consider asking on a support forum site.
+- A site design concern, create an issue at [MicrosoftDocs/Feedback](https://github.com/MicrosoftDocs/Feedback/Issues/new/choose).
+- A problem completing a tutorial, compare your code with the completed sample.
+- A duplicate of an open or closed issue, leave a comment on that issue.
+
+**Issue description**
+
+&lt;include description here&gt;
+
+**Target framework**
```

Comment:
More than Target Framework is asked for in this section.  Could be named "Environment" or "Software versions"?  I wonder how often it will get filled out and will be helpful.  Perhaps we could plan a review after some months, and if it doesn't get used much, remove the second two sections leaving only "Before you open an issue".